rotkehlchen/api/v1/schemas.py

```diff
@@ -652,7 +654,10 @@ def make_history_event_filter(
             data: dict[str, Any],
             **_kwargs: Any,
     ) -> dict[str, Any]:
-        should_query_evm_event = any(data[x] is not None for x in ('evm_chain', 'products', 'counterparties'))  # noqa: E501
+        should_query_evm_event = (
```

Comment:
no this is not correct as logic. Ethereum staking events are **also** in ethereum. We can't use this logic. I think you need to keep the evm chain argument as a logic and translate it here to `data['location'] in EVM_LOCATIONS and data['event_identifier'].startswith('0x')` or something along those lines.